docs/source/quick_start.rst

```diff
@@ -31,6 +31,8 @@ Creating Credentials and Sources for a Network
 
 A *network* source is composed of one or more host names, one or more IP addresses, IP ranges, or a combination of these network resources. In addition, the source creation command references one or more credentials. Typically, a network source might include multiple credentials because it is expected that many credentials would be needed to access a broad IP range.
 
+**IMPORTANT:** For a network scan, Quipucords must use the credentials to run some commands with sudo access. For more information about this requirement, see `Commands Used in Scans of Remote Network Assets <commands.html>`_
+
```

Comment:
Nitpick: the ever present threat that someone is using one of the other "become" methods that is not `sudo`
`... to run some commands that require escalated privileges. This is typically provided by use of sudo. For more information ...`

Reply:
"sudo access" changed to "elevated privileges"